server/api/topics.js

```diff
@@ -31,11 +31,12 @@ router.get("/:topic_id", passport.authenticate("jwt", { session: false }),(req,
 
 
 router.post("/create", function(req, res) {
-	console.log("work-create");
-
 	const newTopicName = req.body.topic_name;
 	const documentName = req.body.document_name;
 	const documentLink = req.body.document_link;
```

Comment:
I suppose if these are not required fields, right? So if they are undefined are you sure that the request is not failing?

Reply:
Done  
made a change on recreating-schema